app/hydrator/services/create/stores/config-store.js

```diff
@@ -202,7 +207,7 @@ class HydratorPlusPlusConfigStore {
           _backendProperties: node._backendProperties,
         },
         information: node.information,
-        outputSchema: node.outputSchema,
+        outputSchema: angular.isArray(node.outputSchema) ? node.outputSchema[0].schema : node.outputSchema,
```

Comment:
why use angular.isArray and not Array.isArray?

Reply:
There are other places using this so I borrowed as example